main/src/ca/uwaterloo/flix/tools/Packager.scala

```diff
@@ -264,9 +266,10 @@ object Packager {
       addToZip(zip, "META-INF/MANIFEST.MF", manifest.getBytes)
 
       // Add all class files.
-      for (buildFile <- getAllFiles(getBuildDirectory(p))) {
-        val fileName = getBuildDirectory(p).relativize(buildFile).toString
-        val fileNameWithSlashes = fileName.replace('\\', '/')
+      // Here we sort entries by relative file name to apply https://reproducible-builds.org/
+      for ((buildFile, fileNameWithSlashes) <- getAllFiles(getBuildDirectory(p))
+          .map{(path: Path)=>(path, getBuildDirectory(p).relativize(path).toString.replace('\\', '/'))}
```

Comment:
Lets introduce a helper function for `getBuildDirectory(p).relativize(path).toString.replace('\\', '/'))`.

You don't have to write `p{(path: Path)=>(` it should work with just `p(path => ...`.